websites/M/Modrinth/presence.ts

```diff
@@ -8,17 +8,17 @@ presence.on("UpdateData", async () => {
 			largeImageKey: "https://i.imgur.com/GnMSCG4.png",
 			startTimestamp: browsingTimestamp,
 		},
-		{ pathname, hostname, href } = document.location,
+		{pathname, hostname, href} = document.location,
 		pathSplit = pathname.split("/").filter(x => x),
 		active = document.querySelector('[class*="is-active"]');
 
 	switch (hostname) {
 		case "modrinth.com": {
 			if (active?.textContent) {
-presenceData.details = `Browsing ${
+				presenceData.details = `Browsing ${
 					pathSplit[0]
 				} ${active?.textContent.toLowerCase()}`;
-}
+			}
```

Comment:
Without the else or returning it won't work and will just go to plugin in the switch statement.
So Option 1 is providing an else
or Option 2:
``		presence.setActivity(presenceData);
		return;``

Reply:
so i need to replace line 185-186 with
```js
	presence.setActivity(presenceData);
	return;
```

Reply:
> so i need to replace line 185-186 with
> 
> ```js
> 	presence.setActivity(presenceData);
> 	return;
> ```

No